doc/source/cluster/kubernetes/getting-started.ipynb

```diff
@@ -244,6 +258,163 @@
     "Note that in production scenarios, you will want to use larger Ray pods. In fact, it is advantageous to size each Ray pod to take up an entire Kubernetes node. See the [configuration guide](kuberay-config) for more details."
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "ef55f61f",
+   "metadata": {},
+   "source": [
+    "### Deploying a static Ray cluster\n",
+    "\n",
+    "\n",
+    "#### Deploying a static Ray cluster without KubeRay\n",
+    "\n",
+    "In this section, we will deploy a static Ray cluster into the default namespace without using KubeRay. To use a non-default namespace, specify the namespace in your kubectl commands:\n",
+    "\n",
+    "`kubectl -n <your-namespace> ...`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3a8267f1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Deploy a sample Ray Cluster from the Ray repo:\n",
+    "! kubectl apply -f https://raw.githubusercontent.com/ray-project/ray/master/doc/source/cluster/kubernetes/configs/static-ray-cluster.with-fault-tolerance.yaml\n",
```

Comment:
No, we need to first set the real password then run this command.

Reply:
I see– could we change the default `redis-password` in the config to be either empty or `5241590000000000` (Ray's default Redis password)? And then could we add a comment or a note saying the if the command fails, the user should download the config file, change the password, and retry?

Ideally, this command should work out-of-the-box since it'll be confusing if it always fails.

Reply:
Done, I set that as an empty string because it is supposed to be the password for the external Redis rather than the default Ray Redis.

Reply:
Sounds good!